extra/1-currency-conversion.js

```diff
@@ -5,7 +5,10 @@
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
 
-function convertToUSD() {}
+function convertToUSD(pound) {
+  let usd=Math.round(pound*1.4*100)/100; 
```

Comment:
This confused me to begin with as to why you were multiplying pounds by 1.4 and 100. One of the reasons is that I have no idea what the significance of the numbers. In the industry we'd refer to these as "magic numbers". It's considered better to treat them as named constants defined somewhere in your code. Some people/organisations will capitalize shared constants, something like:
```
const EXCHANGE_RATE_GBP_USD = 1.4;

function convertToUSD(pound) {
    return Math.round(pound * EXCHANGE_RATE_GBP_USD);
}
```

Reply:
@jdbevan, 
Yes you are right, it is confusing, but what I understood the question is asking to return the number into 2-decimal places. I know we can using toFixed function but it returns string instead of number. Therefore, I used Maths technique by multiplying the number by 100 then rounded (to make it integer). Then again divide the new integer by 100 to get a number with 2-deceimal places.